### ISSUE TITLE
Community to discuss with other users

### ISSUE THREAD

**Issue by Lanhild** [louislam/uptime-kuma#4626]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Is there any exisiting community to discuss with other users of Uptime Kuma?

We could have either GitHub discussions or a Discord/Slack or the likes to discuss with other users about Uptime Kuma.

That'd be more convenient that opening issues.

### 📝 Error Message(s) or Log

n/a

### 🐻 Uptime-Kuma Version

n/a

### 💻 Operating System and Arch

n/a

### 🌐 Browser

n/a

### 🖥️ Deployment Environment

n/a

**Comment by CommanderStorm** [comment]
We have tried github discussions in the past. Please see
- https://github.com/louislam/uptime-kuma/issues/4191
- https://github.com/louislam/uptime-kuma/issues/92
- https://github.com/louislam/uptime-kuma/issues/2579

For the suggestion of a Discord servers, please see https://github.com/louislam/uptime-kuma/issues/3770 
I don't think anything has changed since September.
TLDR: Discord is non-publicly-indexable, meaning that we can't use the positive network effects of helping out in the open and being indexable by Google/GitHub search.

For the suggestion of Slack, next to being not indexable, this is also 10€/User.

Please see https://www.reddit.com/r/UptimeKuma/ for a reddit page which is not monitored by the core devs like github for capacity reasons.
